backend/frontend/views.py

```diff
@@ -274,7 +294,7 @@ def action_start_card(request, card_id):
     )
 
 
-@user_passes_test(is_super)
+@login_required()
 def users_and_teams_nav(request):
```

Comment:
Do we not need to make this view filter the allowed teams as well? Or maybe it's a different task?

Reply:
I don't think there's a way to access the `team_id.` for this view from the can_view_team method.

Besides, the view is not responsible for fetching any teams. The "sub-view" `view_partial_teams_list` filters only the teams a user is allowed to see.

At least that's how I understand everything :see_no_evil: